root/etc/e-smith/events/actions/nethserver-directory-user-delete

```diff
@@ -36,14 +36,7 @@ if ($userName =~ /@/) {
     $userName = (split(/@/,$userName))[0];
 }
 
-my $home = (getpwnam($userName))[7];
-
-my $opt = "";
-if ( -d $home ) {
-   # remove home directory if exists
-   $opt = "-r";
-}
-my $discard = `/usr/sbin/luserdel $opt '$userName'`;
+my $discard = `/usr/sbin/luserdel '$userName'`;
```

Comment:
```suggestion
system('/usr/sbin/luserdel', $userName);
```

Reply:
good but in this case we should retrieve the exit code and warn or die in case or fail

Reply:
IIRC `$?` works like Bash here. Please verify it!

Reply:
`system (/path/to/bash/command) == 0 || die 'this command has died';`

Reply:
I guess if `$!` should not be better rather than `$?`